src/pollers/gateway.rs

```diff
@@ -164,6 +164,19 @@ impl Default for RetryConfig {
     }
 }
 
+impl RetryConfig {
+    pub(crate) fn poll_retry_policy() -> Self {
```

Comment:
Wonder if this value should be static/global and just use references of it when needed? Does it ever get mutated? May have to change the next part to `impl From<&RetryConfig> for ExponentialBackoff` or something.

Reply:
There's kinda no point since a new owned instance needs to be recreated every time. We'd end up just cloning the static instance.

Reply:
It's also like ~48 bytes

Reply:
> There's kinda no point since a new owned instance needs to be recreated every time

Ah, ok, haven't looked at where all it is sent. Granted I understand that for exponential backoff since it's mutable. I suppose a `Cow` type could be used for this shared type, but if not seen as worth it, no prob. Wasn't too concerned w/ size as much as instantiation over and over, but I understand if that's over-optimization.